index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b08094b2a2.js" crossorigin="anonymous"></script>
+    <title>Personal Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav><i class="fas fa-bars"></i></nav>
+    </header>
+    <section>
+        <h1 class="header">Hey there.
+            <br>I’m Alex.
+           <h1 class="text_color">I’m a software developer</h1>
```

Comment:
You did great at removing the h2 within the h1 element. However, h1 element does not have to be within h1 element either. Kindly remove the inner tag.

Reply:
@emmiiorji what do you suggest I use instead since they have different styles?

Reply:
Hi @NgachaIan,

I think you can enclose each with a span element.